src/server.js

```diff
@@ -1,7 +1,7 @@
 import express from "express";
 import handlebars from "handlebars";
 import fs from "fs";
-import presentations from './db/presentations.json' with {type: "json"}
+import presentations from './db/presentations.json' assert {type: "json"}
```

Comment:
Why the change to assert here? Will an error occur if the file you're importing is not json?

Reply:
i think it wasn't working for me unless I has assert, might be a versioning issue. I can change it back!